### PR TITLE
Fix not displaying mutant + cbm encumbrance

### LINKED ISSUE #23928: Mutations with encumbrance_always don't show increased encumbrance in ENCUMBRANCE AND WARMTH menu
Game version: 0.C-7435

Operating system:Windows XP SP3 x86

Tiles or curses: curses

Mods active: default only

### Expected behavior
Mutations with `encumbrance_always` such as Slit Nostrils should show increased encumbrance in `ENCUMBRANCE AND WARMTH` menu.

### Actual behavior

default

default

### Steps to reproduce the behavior
Remember your current mouth encumbrance, get yourself Slit Nostrils, look at the mouth encumbrance again. 

I feel that increased encumbrance is actually applied, it's just menu tha doesn't show new values.

### PR DESCRIPTION
**Pull request by Ramza13**
#### Summary
SUMMARY: Bugfixes "Fixed display to show mutation and cbm encumbrance"

#### Purpose of change
Fixes  #23928

#### Describe the solution
Changed encumbrance to add mutation and cbm encumbrance.  Previously was showing only armor encumbrance in first number now is showing total minus layering.

### CODE REVIEW

File: src/player_display.cpp [hunk 1]
```diff
@@ -127,7 +127,7 @@ void player::print_encumbrance( const catacurses::window &win, const int line,
                               ( highlighted ? c_green : c_light_gray );
         mvwprintz( win, point( 1, 1 + i ), limb_color, "%s", out );
         // accumulated encumbrance from clothing, plus extra encumbrance from layering
-        mvwprintz( win, point( 8, 1 + i ), encumb_color( e.encumbrance ), "%3d", e.armor_encumbrance );
+        mvwprintz( win, point( 8, 1 + i ), encumb_color( e.encumbrance ), "%3d", e.encumbrance - e.layer_penalty );
```
Comment (kevingranade) [inline src/player_display.cpp]:
```suggestion
        mvwprintz( win, point( 8, 1 + i ), encumb_color( e.encumbrance ), "%3d",
                   e.encumbrance - e.layer_penalty );
```

**Comment by kevingranade** [comment]
The comment specifically says ``accumulated encumbrance from clothing, plus extra encumbrance from layering``, why did you decide to back out the layering penalty?

**Comment by Ramza13** [comment]
> 
> 
> The comment specifically says `accumulated encumbrance from clothing, plus extra encumbrance from layering`, why did you decide to back out the layering penalty?

Because the comment was wrong it used to just show armor_encumbrance which does not include layering penalty.  Also to me the plus sign between the two numbers indicated that you had to add them to get the total encumbrance so it made more sense this way.

While trying to fix this the first solution I tried was to either list just the total or the armor encumbrance, cbm encumbrance, mutant encumbrance and layer penalty with an option to change between them.  I also tried total encumbrance and layer penalty with changing the plus sign to a |. I then decided these were overkill and it made more sense to just work how I had assumed it worked.

That said I don't mind changing it if it makes more sense the other way.

**Comment by kevingranade** [comment]
OK that's cool then.

**Pull request merged**